### ISSUE TITLE
Add prioritisation of issues using tap gestures

### ISSUE THREAD

**Issue by nikhilaravi** [dwyl/tudo#92]
Use tap gestures to get people to go through their issues and add priority labels v quickly: 

e.g. Priority 1 = low, Priority 2 = high, 

tap once/single click = Priority 1
tap twice/double click = Priority 2
etc

@nelsonic suggested it should take more effort to create high priority items  

This will enable ordering of issues v quickly without manually adding labels to each one. 

Any other ideas??

**Comment by tsop14** [comment]
Where would they click? Eventually clicking on an issue will open up its single page in app - could this be confusing? But good idea if there was a clear place to click :dancer:

**Comment by nelsonic** [comment]
Agreed with @tsop14 Tap gestures need to have a `click` fallback. but love the idea of a simple interaction. 
relates to: #93

**Comment by iteles** [comment]
:+1: I think that ultimately this will be resolved by whether there are other interactions that need to happen more often (daily for example) that also need to happen quickly. Love this idea though <3
